tests/Facebook/InstantArticles/Transformer/TransformerTest.php

```diff
@@ -34,6 +35,13 @@ public function testTransformString()
         $this->assertEqualsHtml('<h1>Title String</h1>', $header->getTitle()->render());
     }
 
+    public function testDontTransformHTMLEntitiesTwice()
+    {
+        $author = Author::create()->withName("Test &amp; Test");
```

Comment:
We need another test case here, just to be sure we are doing it correctly with the &amp;

`Author::create()->withName("Test & Test")`
and
`Author::create()->withName("Test & Test")->withURL('http://facebook.com/param1=val1&param2=val2')`

Reply:
Ok I will merge this and create an issue to add the tests

Reply:
#297